client/dashboard/components/snackbar-back-button/index.tsx

```diff
@@ -16,6 +16,8 @@ export function getSnackbarBackButtonText(
 			return __( 'Back to Site Deployments' );
 		case 'site-domains':
 			return __( 'Back to Site Domain Names' );
+		case 'site-settings-sftp-ssh':
```

Comment:
I initially had the copy "Back to Site Settings > SFTP/SSH" but the component has a chevron, so "< Back to Site Settings > SFTP/SSH" didn't look good to me.